test/functional/Valhalla/playlist.xml

```diff
@@ -63,13 +63,13 @@
 		<variations>
 			<variation>-Xjit:count=0</variation>
 			<variation>-Xjit:count=1,disableAsyncCompilation -Xgcpolicy:optthruput</variation>
-			<variation>-Xjit:count=1,disableAsyncCompilation -Xgcpolicy:optthruput -XX:ValueTypeFlatteningThreshold=99999 -XX:-EnableArrayFlattening</variation>
+			<!--<variation>-Xjit:count=1,disableAsyncCompilation -Xgcpolicy:optthruput -XX:ValueTypeFlatteningThreshold=99999 -XX:-EnableArrayFlattening</variation>
```

Comment:
Please include the JIT issue number as a comment here.

Reply:
Its there now.

Reply:
FYI @hzongaro @a7ehuo Some JIT variations of this test are being disabled because of so many failures.